src/config/backup.php

```diff
@@ -0,0 +1,159 @@
+<?php
+
+return [
+
+    'backup' => [
+
+        /*
+         * The name of this application. You can use this name to monitor
+         * the backups.
+         */
+        'name' => env('APP_URL'),
+
+        'source' => [
+
+            'files' => [
+
+                /*
+                 * The list of directories that should be part of the backup. You can
+                 * specify individual files as well.
+                 */
+                'include' => [
+                    base_path(),
+                ],
+
+                /*
+                 * These directories will be excluded from the backup.
+                 * You can specify individual files as well.
+                 */
+                'exclude' => [
+                    base_path('vendor'),
```

Comment:
Why not `node_modules` too? See https://docs.spatie.be/laravel-backup/v5/installation-and-setup

Reply:
I think we could just not publish the config file ourselves.
spatie package has autodiscovery.

we could just add this line to README.

```
php artisan vendor:publish --provider=" Spatie\Backup\BackupServiceProvider"
```

This way we will not have to mantain our config file to mimic theirs.

What do you think?

Reply:
So long as it goes into the documentation for backpack - @tabacitu.